syntax = "proto3";

message PersonMsg {
    int32 id = 1;
    string first_name = 2;
    string last_name = 3;
    string company_name = 4;
}

message LocationMsg {
    int32 id = 1;
    int32 person_id = 2;
    string latitude = 3;
    string longitude = 4;
    string creation_time = 5;
}

message Person {
    bool person = 1;
}

message Location {
    bool location = 1;
}

service CallService {
    rpc create_person(PersonMsg) returns (Person);
    rpc create_loc(LocationMsg) returns (Location);
}